.readthedocs.yml

```diff
@@ -1,4 +1,20 @@
-requirements_file: requirements.txt
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+version: 2
 
+sphinx:
```

Comment:
we don't seem to have this section in any of our production config, but I suppose it won't hurt to list these defaults

Reply:
Some of these are currently set in the RTD admin panel, but I think RTD devs wanna move away from that and define everything in the YAML eventually. Also being explicit doesn't hurt, as you said.